ote/ote/modules/arg_converters/mmpose.py

```diff
@@ -0,0 +1,33 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from .mmdetection import MMDetectionArgConverterMap
+from ..registry import ARG_CONVERTER_MAPS
+
+# FIXME. Remove?
```

Comment:
you can use MMDetectionArgConverterMap if you modify your training dataset so that it has nested RepeatDataset section (with times=1)